roles/cluster/cluster_deploy_operator/tasks/main.yml

```diff
@@ -276,33 +277,32 @@
     fail: msg="ClusterServiceVersion install not successful ({{ operator_csv_phase.stdout }})"
     when: operator_csv_phase.stdout != "Succeeded"
 
-  always:
-  - name: Store the YAML of the operator CSV that was installed (debug)
-    shell:
-      oc get ClusterServiceVersion/{{ operator_csv_name }}
-         -oyaml
-          -n "{{ cluster_deploy_operator_namespace }}"
-          > {{ artifact_extra_logs_dir }}/operator_csv.yml
-  - name: Store the YAML of the subscription
-    shell:
-      oc get -f "{{ artifact_extra_logs_dir }}/src/002_sub.yml"
-         -oyaml
-         -n "{{ cluster_deploy_operator_namespace }}"
-         > {{ artifact_extra_logs_dir }}/operator_sub.yml
   rescue:
   - name: Capture the Catalog Operator logs (debug)
-    shell:
-      oc logs deployment.apps/catalog-operator
-         -n openshift-operator-lifecycle-manager
-         > {{ artifact_extra_logs_dir }}/catalog_operator.log
+    shell: |
+      oc logs deployment.apps/catalog-operator \
+        -n openshift-operator-lifecycle-manager \
+        > {{ artifact_extra_logs_dir }}/catalog_operator.log
     failed_when: false
-
   - name: Indicate where the Catalog-operator logs have been saved
     debug: msg="The logs of Catalog Operator have been saved in {{ artifact_extra_logs_dir }}/catalog_operator.log"
-
   - name: Failed because the operator could not be installed from the CatalogSource
     fail: msg="Failed because the operator could not be installed from the CatalogSource"
 
+  always:
+  - name: Store the YAML of the operator CSV that was installed (debug)
+    shell: |
+      oc get ClusterServiceVersion/{{ operator_csv_name }} \
+         -oyaml \
+          -n "{{ cluster_deploy_operator_namespace }}" \
+          > {{ artifact_extra_logs_dir }}/operator_csv.yml
+  - name: Store the YAML of the subscription
```

Comment:
could you add an empty line here (it wasn't there before, but while we're at it ;-) )